tap_github/streams.py

```diff
@@ -904,3 +904,44 @@ def http_headers(self) -> dict:
             ),
         ),
     ).to_dict()
+
+
+class ContributorsStream(GitHubStream):
+    """Defines 'Contributors' stream."""
+
+    name = "contributors"
+    path = "/repos/{org}/{repo}/contributors"
+    # primary_keys = ["id"]
```

Comment:
```suggestion
    primary_keys = ["org", "repo", "node_id"]
```
Using `id` (which is the user id) leads to a bug where a single user can only have contributor data for one repo. In other words, if I've contributed to 2 repos for which I'm fetching this stream, I will only get 1 record for my contributions to one of them, the other record is missing in the output. The change above fixes that.

Reply:
We do still have the issue of the anonymous tap. One option I was thinking about was to split the anonymous contributors into another stream AnonymousContributorStream and filter it to ONLY grab anonymous contribution and the associated fields. What do you think?